deployment/legacy-migration/maintenance-worker/migration/neo4j/badges.cql

```diff
@@ -45,7 +45,7 @@ MERGE(b:Badge {id: badge._id["$oid"]})
 ON CREATE SET
 b.key       = badge.key,
 b.type      = badge.type,
-b.icon      = badge.image.path,
+b.icon      = substring(badge.image.path, 38),
```

Comment:
@ulfgebhardt I find it quite dangerous, how do you know that it's always the same first part of the URL?

Reply:
Use [replace](https://neo4j.com/docs/cypher-manual/current/functions/string/#functions-replace).